include/fmt/core.h

```diff
@@ -43,6 +43,17 @@
 #  define FMT_HAS_GXX_CXX11 0
 #endif
 
+#if __cplusplus >= 201703L
+#  ifndef __GLIBCXX__
+#    define FMT_HAS_CONSTEXPR_CHAR_TRAITS 1
+#  elif defined(_GLIBCXX_RELEASE) && _GLIBCXX_RELEASE >= 7
+#    define FMT_HAS_CONSTEXPR_CHAR_TRAITS 1
+#  endif
+#endif
+#ifndef FMT_HAS_CONSTEXPR_CHAR_TRAITS
+#  define FMT_HAS_CONSTEXPR_CHAR_TRAITS 0
+#endif
```

Comment:
I suggest you comment on this check because it's not a really straightforward one 🙂